src/uint256.h

```diff
@@ -20,10 +20,11 @@ class base_blob
     static constexpr int WIDTH = BITS / 8;
     uint8_t m_data[WIDTH];
 public:
-    base_blob()
-    {
-        memset(m_data, 0, sizeof(m_data));
-    }
+    /* construct 0 value by default */
+    base_blob() : m_data() {}
+
+    /* constructor for constants between 1 and 255 */
+    explicit constexpr base_blob(uint8_t v) : m_data{v} {}
```

Comment:
```suggestion
    constexpr explicit base_blob(uint8_t v) : m_data{v} {}
```

style nit: Would be nice to use the same ordering of keywords for different lines in the same patch. 

<!--
(https://en.cppreference.com/w/cpp/utility/optional/optional)

Reply:
Makes sense; https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#formatting-declarations-and-definitions also suggests `constexpr explicit`